exporter/sapmexporter/README.md

```diff
@@ -32,6 +33,16 @@ during final translation.  Intended to be used in tandem with identical configur
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `correlation`
+  - `enabled` (default = true): Whether to enable span/metric correlation.
+  - `endpoint` (default = ""): If set this is the endpoint where correlation API calls will be made. Otherwise it defaults to using `api_endpoint` value.
+  - `stale_service_timeout` (default = 5 minutes): How long to wait after a span's service name is last seen before uncorrelating it.
+  - `max_requests` (default = 20): Max HTTP requests to be made in parallel.
+  - `max_buffered` (default = 10,000): Max number of correlation updates that can be buffered before updates are dropped.
+  - `max_retries` (default = 2): Max number of retries that will be made for failed correlation updates.
+  - `log_dimension_updates` (default = false): Whether or not to log correlation updates to dimensions (at DEBUG level).
```

Comment:
I think the notion of _dimension updates_ are too specific given that the API is meant to abstract that and potentially reimplement it later.